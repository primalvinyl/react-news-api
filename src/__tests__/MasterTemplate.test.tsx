import React from 'react';
import { shallow } from 'enzyme';
import Renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import MasterTemplate from '../components/_MasterTemplate';

describe('MasterTemplate', () => {
    it('renders', () => {
        const myWrapper = shallow(
            <MemoryRouter>
                <MasterTemplate>
                    <div>Test</div>
                </MasterTemplate>
            </MemoryRouter>
        );
        expect(myWrapper.exists());
    });
    it('matches snapshot', () => {
        const component = Renderer.create(
            <MemoryRouter>
                <MasterTemplate>
                    <div>Test</div>
                </MasterTemplate>
            </MemoryRouter>
        );
        expect(component.toJSON()).toMatchSnapshot();
    });
});
