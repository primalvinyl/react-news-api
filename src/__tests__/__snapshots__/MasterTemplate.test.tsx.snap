// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MasterTemplate matches snapshot 1`] = `
<div
  className="root"
>
  <div>
    Test
  </div>
</div>
`;
